primitives/runtime/src/storage_keys.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
This whole file is: (1) wrong, as all our pallets (and `paras` pallet) are based on frame v2, which means that the storage keys are computed slightly differently (2) obsolete, because in master there are **correct** versions of these functions declared at the crate level.

I could fix it in this PR, or in follow-up PRs.

Reply:
fixed